metau-capstone/LoginViewController.m

```diff
@@ -0,0 +1,67 @@
+//
+//  LoginViewController.m
+//  metau-capstone
+//
+//  Created by Shirley Zhu on 7/6/22.
+//
+
+#import "LoginViewController.h"
+#import "Parse/Parse.h"
+
+@interface LoginViewController ()
+
+@end
+
+@implementation LoginViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    // Do any additional setup after loading the view.
+}
+- (IBAction)loginUser:(id)sender {
+    NSString *username = self.usernameField.text;
+    NSString *password = self.passwordField.text;
+    
+    [PFUser logInWithUsernameInBackground:username password:password block:^(PFUser * user, NSError *  error) {
+        if (error != nil) {
+            NSLog(@"User log in failed: %@", error.localizedDescription);
+        } else {
+            NSLog(@"User logged in successfully");
+            
+            // display view controller that needs to shown after successful login
+        }
+    }];
+}
+
+- (IBAction)registerUser:(id)sender {
+    // initialize a user object
+    PFUser *newUser = [PFUser user];
```

Comment:
nit: const

eg.
```
PFUser *const newUser = [PFUser user];

Reply:
Okay! Why do I want to use const here?

Reply:
This is a coding standard, if any local variable is intended not to be reassigned, annotate it with const would safeguard it to be unintentionally reassigned later on